acquia_lift.batch.inc

```diff
@@ -10,11 +10,12 @@
  * Batch syncs all campaigns to Lift.
  */
 function acquia_lift_batch_sync_campaigns($agents = array()) {
-  // First delete the queue because we are about to sync everything now.
-  acquia_lift_delete_queue();
-  // Now get all sync operations for all existing Acquia Lift agents.
+  // Now get all sync operations for all existing Acquia Lift testing agents.
   if (empty($agents)) {
     foreach (acquia_lift_get_agent_types() as $type => $info) {
+      if ($type == 'acquia_lift_target') {
```

Comment:
Was there a reason to not use acquia_lift_is_testing_agent() or did you just think the function call overhead wasn't worth it?